Core/StitchCoreSDK/Tests/StitchCoreSDKTests/StitchAppClientInfoUnitTests.swift

```diff
@@ -2,7 +2,21 @@ import Foundation
 import XCTest
 @testable import StitchCoreSDK
 
-class StitchAppClientInfoUnitTests: XCTestCase {
+class StitchAppClientInfoUnitTests: XCTestCase, NetworkMonitor, AuthMonitor {
+    var state: NetworkState = .connected
+
+    var isLoggedIn: Bool = true
+
+    var isConnected: Bool = true
+
+    func add(networkStateDelegate delegate: NetworkStateDelegate) {
+
```

Comment:
[q] should this be a no-op or should we `XCTFail`?

Reply:
I think no-op is more appropriate here.